app/controllers/application_controller.rb

```diff
@@ -206,13 +206,13 @@ def web_timeout(&block)
     if e.is_a?(Timeout::Error) ||
        (e.is_a?(ActiveRecord::StatementInvalid) && e.message.include?("execution expired"))
       ActiveRecord::Base.connection.raw_connection.cancel
-      render :action => "timeout"
+      render "browse/timeout", :layout => map_layout
```

Comment:
I know you change it again later but I don't think it's reasonable to have this as an intermediate step as it leaves the site broken - the while point of only setting an action was to render the timeout action for the current controller because timeouts can occur in different controllers.

I think this needs more though about how to do this change and the following one to ensure we keep things roughly working through each step.

Reply:
The site is broken before the first step. How can it work through each step unless you squash all of the steps?

Reply:
Removed temporary `render "browse/timeout"`.

Reply:
There are two controllers (changesets and changeset_comments) which have a timeout view already and with this change they will now render the browse timeout instead of their own one if they timeout? That was what I meant by breaking things that currently worked.